images/Dockerfile.ubuntu

```diff
@@ -80,6 +75,61 @@ RUN apt-get update \
     zstd \
     && apt-get remove -y unattended-upgrades && apt-get clean && rm -rf /var/lib/apt/lists/*
 
+# Packages and scripts only for specific images
+FROM common AS ubuntu-specific
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+    dbus-user-session \
+    dracut-live \
+    firmware-sof-signed \
+    fuse3 \
+    krb5-locales \
+    linux-image-generic-hwe-22.04 \
+    pigz \
+    pkg-config \
+    polkitd \
+    systemd-hwe-hwdb \
+    systemd-resolved \
+    && apt-get remove -y unattended-upgrades && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+FROM common AS ubuntu-20-lts-specific
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+    file \
+    fuse \
+    krb5-locales \
+    linux-image-generic-hwe-20.04 \
+    lldpd \
+    patch \
+    policykit-1 \
+    snmpd \
+    ubuntu-advantage-tools \
+    && apt-get remove -y unattended-upgrades && apt-get clean && rm -rf /var/lib/apt/lists/*
+# workaround https://github.com/kairos-io/kairos/issues/949
+# TODO: backport patch into packages
+COPY images/dracut-broken-iscsi-ubuntu-20.patch /
+RUN cd /usr/lib/dracut/modules.d/95iscsi && patch < /dracut-broken-iscsi-ubuntu-20.patch && rm -rf /dracut-broken-iscsi-ubuntu-20.patch
```

Comment:
this part here is actually not required any more, that's how I spotted the inconsistency. Will remove after review, just want to use this to exemplify the benefit of having a multi-stage setup